src/server/rest/v1/service/UserService.ts

```diff
@@ -226,6 +226,8 @@ export default class UserService {
       if (filteredRequest.role) {
         await UserStorage.saveUserRole(req.tenant, user.id, filteredRequest.role);
       }
+      // Save Technical user flag
+      await UserStorage.saveUserTechnicalFlag(req.tenant, user.id, user.technical);
```

Comment:
No need a dedicated method for this flag.
I think that only Admins can see and check this flag, then this should be treated like Admin data (check UserStorage.saveUserAdminData and plateID prop and mimic it)